frontend-react/src/hooks/UseOrganizationReceivers.ts

```diff
@@ -25,11 +26,20 @@ export const useOrganizationReceivers = () => {
         }
         return null;
     }, [isAdmin, authorizedFetch, parsedName]);
+    const { data, isLoading } = useSuspenseQuery({
+        queryKey: [receivers.queryKey, activeMembership],
+        queryFn: memoizedDataFetch,
+    });
+    const allReceivers = (data ?? []).sort((a, b) =>
+        a.name.localeCompare(b.name),
+    );
+    const activeReceivers = allReceivers.filter(
+        (receiver) => receiver.customerStatus !== CustomerStatusType.INACTIVE,
+    );
     return {
-        ...useSuspenseQuery({
-            queryKey: [receivers.queryKey, activeMembership],
-            queryFn: memoizedDataFetch,
-        }),
+        allReceivers: allReceivers,
```

Comment:
The return here should be everything from useSuspenseQuery + anything custom (the pattern being always expecting the return of useSuspenseQuery across all hooks)

Reply:
Got it! Changed.